test/e2e/container_inspect_test.go

```diff
@@ -78,8 +78,28 @@ var _ = Describe("Podman container inspect", func() {
 		Expect(session).Should(ExitCleanly())
 
 		data := podmanTest.InspectContainer(ctr2)
-		Expect(data).To(HaveLen(1))
```

Comment:
This should not be removed.

Reply:
Oops, this should not have been committed.
While at it, I found no way to pass the secret value with stdin, so I used a file. attribute. Do know a way ?